CHANGELOG.md

```diff
@@ -28,25 +41,25 @@
 
 ## 1.0.0
 
-*   Huge performance boost (almost 10 times faster when zooming or panning)
```

Comment:
change unrelated to the subject of your PR, please revert

Reply:
This change was automatically done by husky, so I assume it should be there?

Reply:
you're right, let's keep your change.